src/api/userApi.js

```diff
@@ -0,0 +1,20 @@
+import axiosClient from './axiosClient';
+
+const userApi = {
+	login: (params) => {
+		const url = '/auth/login';
+		return axiosClient
+			.post(url, params)
+			.then((res) => console.log(res))
+			.catch((err) => console.log(err));
```

Comment:
Với khúc ni là cu Hiếu có nghĩ chỉ cần trả lại .post thôi nè, rồi ví dụ khi cu Hiếu gọi hàm ni ra, thì cu Hiếu có thể .then, hoặc là cu Hiếu dùng async await cũng được, tại mình hok có cần xử lý .then .catch ở đây á.
```suggestion
			.post(url, params);
```

Reply:
khúc ni a đợi bên server trả về token hoặc lỗi thì a nghĩ xử lí tiện hơn

Reply:
đúng rồi cu Hiếu, kiểu api là dùng chung á, nên cái ni chỉ trả về dữ liệu thôi nè, còn mình xử lí dữ liệu là mình xử lý ở chỗ khác á, anh nói rứa cu Hiếu hiểu ý anh hok

Reply:
ok3 nhó hehehe